app/scripts/contentscript.js

```diff
@@ -114,7 +114,6 @@ function forwardTrafficBetweenMuxers (channelName, muxA, muxB) {
 
 async function setupPublicApi (outStream) {
   const api = {
-    forceReloadSite: (cb) => cb(null, forceReloadSite()),
```

Comment:
Does removing this interfere with our network-switching reloads?

Reply:
Nope, this is only undoing parts of #6904